projects/npm-tools/packages/npm-scripts/src/scripts/sass.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const path = require('path');
+
+const buildSass = require('../sass/build');
+const getMergedConfig = require('../utils/getMergedConfig');
+const setEnv = require('../utils/setEnv');
+const validateConfig = require('../utils/validateConfig');
+
+const {build: BUILD_CONFIG} = getMergedConfig('npmscripts');
+const CWD = process.cwd();
+
+if (!BUILD_CONFIG) {
+	throw new Error('npmscripts.config.js is missing required "build" key');
+}
```

Comment:
IIRC you can skip this check by writing

```js
const BUILD_CONFIG = getMergedConfig('npmscripts', 'build');
```

instead, due to [this code](https://github.com/liferay/liferay-frontend-projects/blob/b9a5a054c3d91a10f0881a49c130eed9f3e015dc/projects/npm-tools/packages/npm-scripts/src/utils/getMergedConfig.js#L24-L26) which we added in 7b5f5376d0727d2e38fa7bd. It should `throw` if the `build` property is missing.